packages/tomls/src/omnibus/fuzzing.toml

```diff
@@ -156,8 +156,8 @@ usdcUsdMaxStaleDuration = "<%= settings.ONE_MINUTE_IN_SECONDS %>"
 [var.market_1eth]
 market1Eth_riskMatrixIndex = "0"
 market1Eth_maxOpenBaseUnscaled = "37500"
-market1Eth_velocityMultiplierUnscaled = "6.25"
-market1Eth_oracleNodeId = "<%= settings.ethUsdcNodeId %>"
+market1Eth_velocityMultiplierUnscaled = "6.5"
+market1Eth_oracleNodeId = "<%= settings.ethUsdcNodeIdFallbackStorkRedstone %>"
```

Comment:
oracle node IDs in fuzzing are mocked and we don't need them to point to fallback. Could you please revert this?